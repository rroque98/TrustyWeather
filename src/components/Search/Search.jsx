import React from "react";
import "./Search.css";

const Search = ({ placeholderText, searchValue, updateSearchValue }) => (
  <section>
    <input
      className="searchBar"
      type="search"
      placeholder={placeholderText}
      onChange={updateSearchValue}
      value={searchValue}
    ></input>
  </section>
);

export default Search;
