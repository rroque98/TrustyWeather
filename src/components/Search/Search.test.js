import React from "react";
import { render, cleanup } from "@testing-library/react";
import Search from "./index.js";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<Search />);
  expect(asFragment()).toMatchSnapshot();
});
