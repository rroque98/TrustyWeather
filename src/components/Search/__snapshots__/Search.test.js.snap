// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <section>
    <input
      class="searchBar"
      type="search"
      value=""
    />
  </section>
</DocumentFragment>
`;
